api/src/main/resources/messages.properties

```diff
@@ -1,2 +1,3 @@
 ${project.parent.artifactId}.title=Registration Core Module
 ${project.parent.artifactId}.manage=Manage module
+${project.parent.artifactId}.search=MPI Search Patient
```

Comment:
How about rephrasing this to MPI Patient Search?

Reply:
done